Codigo/Declares.bas

```diff
@@ -1969,6 +1969,8 @@ Public Type t_UserFlags
 
     TargetUser As t_UserReference ' Usuario señalado
     
+    AttackedNpc As Integer
```

Comment:
dont keep reference to npc as simple integers, use t_npcref instead

Reply:
si, lo vi, en este caso no afectaba mucho pq es simplemente para q si le pega a otro bicho tenga otro valor, pero lo puedo cambiar.

Reply:
we already keep this value at .flags.NPCAtacado, we should be using that